src/components/Dashboard.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import { Flex, Box } from "@chakra-ui/react";
+import { StockList } from "./StockList";
+import { OrderBook } from "./OrderBook";
+
+export const Dashboard: React.FC = () => {
+  return (
+    <Flex direction={["column", "row"]} height="100vh" overflow="hidden">
+      {/* Sidebar: Stock List */}
+      <Box
+        width={["100%", "25%", "20%"]}
+        p={4}
+        bg="gray.800"
+        color="white"
+        minW="250px"
```

Comment:
it's best to avoid using `px` for height and width, you should use newer CSS features like em or %
https://stackoverflow.com/questions/609517/why-em-instead-of-px